Tests/InaraTests.cs

```diff
@@ -4,6 +4,8 @@
 using System.Collections.Generic;
 using UnitTests;
 
+// Note: Tests work best if the tester has a valid API key configured for Inara.
```

Comment:
That's a bit of a code smell. Tests really need to be 100% independent of user credentials, using mock objects.